src/unistd.rs

```diff
@@ -214,6 +214,36 @@ pub fn write(fd: RawFd, buf: &[u8]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
+pub enum Whence {
+    SeekSet,
+    SeekCur,
+    SeekEnd
+}
+
+impl Whence {
+    fn to_libc_type(&self) -> c_int {
+        match self {
+            &Whence::SeekSet => 0,
+            &Whence::SeekCur => 1,
+            &Whence::SeekEnd => 2
```

Comment:
No, I think the enum is a good thing.  I would just change the returned value to be the constant from libc.